### PR TITLE
feat: update about with explanation SPLASH acronym and aim

### PR DESCRIPTION
**Pull request by abotzki**
- add explanation about SPLASH acronym
- add aims
- add citation

### CODE REVIEW

File: pages/about/about.md [hunk 1]
```diff
@@ -3,22 +3,36 @@ title: About
 ---
 
 ## Who is the ELIXIR SPLASH for?
-The ELIXIR Training SPLASH has been designed to guide life scientists in their efforts to <>. It is based on the various steps of the training lifecycle, although not all the steps will be relevant to everyone.
+The ELIXIR Training SPLASH has been designed to guide trainer in their efforts to plan, design, develop, deliver and evaluate training activties. It is based on the various steps of the training lifecycle, although not all the steps will be relevant to everyone.
```
Comment (mihai-sysbio) [inline pages/about/about.md:6]: Is there a `the` missing?
```suggestion
The ELIXIR Training SPLASH has been designed to guide the trainer in their efforts to plan, design, develop, deliver and evaluate training activties. It is based on the various steps of the training lifecycle, although not all the steps will be relevant to everyone.
```

File: pages/about/about.md [hunk 2]
```diff
@@ -3,22 +3,36 @@ title: About
...
 
-The contents are generated and maintained by the ELIXIR Training platform [ELIXIR-TrP](https://elixir-europe.org/). More information on how we want to sustain and govern ELIXIR Training SPLASH can be found on <>.
+The contents are generated and maintained by the ELIXIR Training platform [ELIXIR-TrP](https://elixir-europe.org/). We want to sustain and govern ELIXIR Training SPLASH via the ELIXIR Programme 2024+.
+
+PSLASH is short for
```
Comment (mihai-sysbio) [inline pages/about/about.md:10]: Typo
```suggestion
SPLASH is short for 
```

File: pages/about/about.md [hunk 3]
```diff
@@ -3,22 +3,36 @@ title: About
...
 ## Funding acknowledgement
-ELIXIR Training SPLASH was developed with funding from BioHackathon2023 and the ELIXIR 2024-2028 programme. We are grateful to all who contributed their time to producing contents for ELIXIR Training SPLASH. 
+ELIXIR Training SPLASH was developed with funding from ELIXIR BioHackathon Europe 2023 and the ELIXIR programme 2024-2028. We are grateful to all who contributed their time to producing contents for ELIXIR Training SPLASH. 
 
 ## How to cite ELIXIR Training SPLASH 
 
 <div class="card bg-light my-4">
   <div class="card-body">
-    <p class="card-text"><i>ELIXIR (2023)</i></p>
+    <p class="card-text"><i>ELIXIR SPLASH Pushing the boundaries of training development (2023)</i></p>
```
Comment (mihai-sysbio) [inline pages/about/about.md:35]: The year here is static, which means it needs manual updating. Could this be replaced with something dynamic that will always look up-to-date?
```suggestion
    <p class="card-text"><i>ELIXIR SPLASH Pushing the boundaries of training development (2023)</i></p>
```

**Review by mihai-sysbio — APPROVED** [review]
Very nice to see these updates, great job @abotzki!

File: pages/about/about.md (hunk 3 above)
Comment (abotzki) [reply pages/about/about.md:35]: I'll open another issue to address this request.

**Pull request merged**